src/features/canvass/components/LocationDialog/pages/HouseholdPage.tsx

```diff
@@ -42,9 +44,11 @@ const HouseholdPage: FC<HouseholdPageProps> = ({
               <Msg id={messageIds.households.single.wasVisited} />
             </Typography>
           )}
-          <Button onClick={onHouseholdVisitStart} variant="contained">
-            <Msg id={messageIds.households.single.logVisitButtonLabel} />
-          </Button>
+          {showLogVisitButton && (
+            <Button onClick={onHouseholdVisitStart} variant="contained">
+              <Msg id={messageIds.households.single.logVisitButtonLabel} />
+            </Button>
+          )}
```

Comment:
I realize now that I mislead you on this. This button should not be hidden for location level assignments. The `useVisitReporting()` hook should handle the reporting separately, but UI wise they should be the same. There should not be a problem showing this button, and allowing household-level reporting, because behind the scenes the hook will handle it by storing the household only in local storage, and reporting to the location.

Reply:
does this mean i should make changes to the hook, since the request fails on location level assignments?